src/services/authService.ts

```diff
@@ -0,0 +1,23 @@
+import { type User } from '@prisma/client'
+import { type User as user } from '../types/user'
+import prisma from '../config/prisma/prisma-client'
+
+export async function login(email: string): Promise<string> {
+  // TODO: Add JWT functionality to user login
+  console.log(email)
+  return 'You Hit the login endpoint'
+
+  // Type Issues with findUnique: https://github.com/prisma/prisma/issues/10903
+
+  // const usr = await prisma.user.findUnique({where:{email:email}})
+  // if (usr){
+  //     return usr.authToken;
+  // } else{
+  //     return "User does not exist in Database";
+  // }
+}
+
+export async function signUp(req: user): Promise<User> {
```

Comment:
i dont think to create user we need totalbadge or name. Maybe we need to create a new Type SignUPType or createUserType with email, password,  username only?

Reply:
I think in this case it is fine, either we can specify the extra things as zero in the req.body (such as the badges), but for a user to be created they should have a name. If we need another type for this I think that should be done in another ticket, but this is the same type that is used to create a user in the seeds as well so it should either be changed for both or not. It might also have to be updated with authTokens are added to users as well (future ticket).